prow/bugzilla/client_test.go

```diff
@@ -1040,27 +1040,28 @@ func TestGetAllClones(t *testing.T) {
 		{
 			"Clones including multiple children",
 			bug1,
```

Comment:
Can we make these tests readable by defining all the bugs and their relationship in the testcases rather than having them reference some bugs with indistinguishable names elsewhere?